app/controllers/catalog_controller.rb

```diff
@@ -114,8 +114,9 @@ def self.chf_search_fields
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
     config.add_index_field solr_name("title", :stored_searchable), label: "Title", itemprop: 'name', if: false
+    config.add_index_field solr_name("additional_title", :stored_searchable), label: "Additional Title", itemprop: 'name', if: :present?
```

Comment:
Actually maybe I'll just leave it as it is, `name` is the closest schema.org, and as far as I know is repeatable. Really don't know how this stuff works, will pay more attention to it when doing the hoped-for future overhaul of the entire search results page. https://project-hydra.slack.com/archives/C0F9JQJDQ/p1490103783658224